spec/spec_helper.rb

```diff
@@ -3,6 +3,11 @@
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
 require 'capybara/rspec'
+require 'capybara/email/rspec'
+
+Capybara.app_host = "http://localhost:3000"
+Capybara.server_host = "localhost"
+Capybara.server_port = "3000"
```

Comment:
I believe capybara runs on localhost already. I would recommend using a port other than 3000 so that you don't have a conflict with your dev server. You should only need to set the port, not the app_host or server_host.